config/initializers/proofer.rb

```diff
@@ -1,2 +1,2 @@
 # gems require UPPER case ENV variables so translate
```

Comment:
`to translate`?

Reply:
It's saying we translate the names to upper case because the gems require them to be like that.

Reply:
I know we had this code before, so it's fine to leave in this PR, but I would like to suggest we eliminate this feature and instead do one of two things:

- Uppercase the env vars explicitly in `application.yml`. Note that Figaro allows you to use lowercase methods with uppercase ENV vars. For example, if `application.yml` had `FOO: 'bar'`, you can access it in the app with `Figaro.env.foo`.
OR
- Call `Figaro.env` in the vendor gems to keep things consistent since the gems are private and are only consumed by our IdP app which is known to use Figaro.

Reply:
:+1: for calling Figaro in the vendor gems. That'll make dev in those gems easier too.